### PR TITLE
Inverted Board

### PR DESCRIPTION
**Pull request by ZurabWeb**
Get Slack to request inverted version of the board for the black player's turn.

Depends on: https://github.com/cjsaylor/chessimage/pull/2

### CODE REVIEW

File: rendering/renderlink.go [hunk 1]
```diff
@@ -37,6 +37,9 @@ func (r RenderLink) CreateLink(gm *game.Game) (*url.URL, error) {
 	q.Add("from", from)
 	q.Add("to", to)
 	q.Add("check", check)
+	if gm.Turn() == "Black" {
```
Comment (cjsaylor) [inline rendering/renderlink.go]: @ZurabWeb I think there is a constant for this value in the chess library we are using.

**Comment by ZurabWeb** [comment]
@cjsaylor Changes in.

**Comment by cjsaylor** [comment]
@ZurabWeb You have a test failing.

**Comment by cjsaylor** [comment]
Closes #29

**Comment by cjsaylor** [comment]
Thanks @ZurabWeb!

**Pull request merged**